Libplanet/Assets/Currency.cs

```diff
@@ -100,6 +128,38 @@ private Currency(SerializationInfo info, StreamingContext context)
             Hash = GetHash();
         }
 
+        /// <summary>
+        /// Gets a fungible asset value with the given <paramref name="quantity"/> of the
+        /// specified <paramref name="currency"/>.
+        /// </summary>
+        /// <param name="currency">The currency to get a value.</param>
+        /// <param name="quantity">The major unit of the fungible asset value,
+        /// i.e., digits <em>before</em> the decimal separator.</param>
+        /// <returns>A fungible asset value with the given <paramref name="quantity"/> of the
+        /// specified <paramref name="currency"/>.</returns>
+        /// <remarks>This cannot specify <see cref="FungibleAssetValue.MinorUnit"/> but only
+        /// <see cref="FungibleAssetValue.MajorUnit"/>.  For more precision, directly use <see
+        /// cref="FungibleAssetValue"/>'s constructors instead.</remarks>
+        [Pure]
+        public static FungibleAssetValue operator *(Currency currency, BigInteger quantity) =>
+            new FungibleAssetValue(currency, majorUnit: quantity, minorUnit: 0);
+
+        /// <summary>
+        /// Gets a fungible asset value with the given <paramref name="quantity"/> of the
+        /// specified <paramref name="currency"/>.
+        /// </summary>
+        /// <param name="quantity">The major unit of the fungible asset value,
+        /// i.e., digits <em>before</em> the decimal separator.</param>
+        /// <param name="currency">The currency to get a value.</param>
+        /// <returns>A fungible asset value with the given <paramref name="quantity"/> of the
+        /// specified <paramref name="currency"/>.</returns>
+        /// <remarks>This cannot specify <see cref="FungibleAssetValue.MinorUnit"/> but only
+        /// <see cref="FungibleAssetValue.MajorUnit"/>.  For more precision, directly use <see
+        /// cref="FungibleAssetValue"/>'s constructors instead.</remarks>
+        [Pure]
+        public static FungibleAssetValue operator *(BigInteger quantity, Currency currency) =>
```

Comment:
Can you please explain why this returns `FungibleAssetValue` instead of `Currency`?

Reply:
Even though this diverts the multiplying operator (`*`), it's not an actual multiplication, but a shortcut for creating a new `FungibleAssetValue`: `123 * foo` is equivalent to `new FungibleAssetValue(currency: foo, major: 1, minor: 0)`.